### PR TITLE
Add list of indices to community index #74

### LINKED ISSUE #74: Add the list of indexes for community to it's index action

### PR DESCRIPTION
**Pull request by grimm**
Fixes #74

### CODE REVIEW

**Comment by grimm** [comment]
@gina-alaska/awrvi  
Ready for review

File: app/views/communities/show.html.haml [hunk 1]
```diff
@@ -14,3 +14,26 @@
...
+      %tr
+        %th Index Value
+        %th Index Version
+        %th View Index
+    %tbody
+      - @community.indices.each do |index|
+        %tr
+          %td
+            Index:
+            = index.awrvi_index.to_f
```
Comment (smacfarlane) [inline app/views/communities/show.html.haml]: Do we want to round this to 2-3 digits?   What do they display in their paper?

**Comment by grimm** [comment]
Looking at the paper it looks like they are displayed with no more than 2 digits.  I will fix.

**Comment by teknofire** [comment]
:+1:

**Comment by smacfarlane** [comment]
:+1:

**Pull request merged**